backend/sozisel/test/sozisel_web/schema/events/quizzes/quiz_queries_test.exs

```diff
@@ -0,0 +1,257 @@
+defmodule SoziselWeb.Schema.Quizzes.QuizQueriesTest do
+  use SoziselWeb.AbsintheCase
+
+  import Sozisel.Factory
+
+  @get_quiz_summary """
+  query GetQuizSummary($id: ID!) {
+    quizSummary(id: $id) {
+        numberOfParticipants
+        averagePoints
+        averageQuizAnswerTime
+    }
+  }
+  """
+
+  @get_quiz_participants_summary """
+  query GetQuizParticipantsSummary($id: ID!) {
+    quizParticipantsSummary(id: $id) {
+        fullName
+        email
+        numberOfPoints
+        quizAnswerTime
+        participantAnswers {
+            questionId
+            finalAnswerIds
+            points
+            trackNodes {
+                reactionTime
+                answerId
+                selected
+            }
+        }
+    }
+  }
+  """
+
+  @get_quiz_questions_summary """
+  query GetQuizQuestionsSummary($id: ID!) {
+    quizQuestionsSummary(id: $id) {
+        questionId
+        question
+        answers {
+            id
+            text
+        }
+        correctAnswers { 
+            id
+            text
+        }
+        averagePoint
+        averageAnswerTime
+        participantsAnswers {
+            fullName
+            email
+            points
+            answerTime
+            finalAnswerIds
+            trackNodes {
+                reactionTime
+                answerId
+                selected
+            }
+        }
+    }
+  }
+  """
+
+  describe "Quiz queries should" do
+    setup do
+      user = insert(:user)
+      [conn: test_conn(user), user: user]
+    end
+
+    test "get quiz summary after summary session", ctx do
+      template = insert(:template)
+      event = insert(:quiz_event, session_template_id: template.id)
+      session = insert(:session, session_template_id: template.id, user_id: ctx.user.id)
+      launched_event = insert(:launched_event, session_id: session.id, event_id: event.id)
+      participant1 = insert(:participant, session_id: session.id)
+      participant2 = insert(:participant, session_id: session.id)
```

Comment:
all those things could be as well in setup function as you use the same set of structures in this test and the one below